backend/app/adapter/routing/handle.go

```diff
@@ -79,3 +81,19 @@ func NewSSOSignInCallback(
 		http.Redirect(w, r, webFrontendURL.String(), http.StatusSeeOther)
 	}
 }
+
+// FeatureHandle retrieves the status of feature toggle.
+func FeatureHandle(
+	instrumentationFactory request.InstrumentationFactory,
+	featureDecisionFactory feature.DecisionFactory,
+) fw.Handle {
+	return func(w http.ResponseWriter, r *http.Request, params fw.Params) {
+		i := instrumentationFactory.NewHTTP(r)
+		featureID := params["featureID"]
+
+		decision := featureDecisionFactory.NewDecision(i)
+		isEnable := decision.IsFeatureEnable(featureID)
+		body, _ := json.Marshal(isEnable)
```

Comment:
I guess that in case of error the body will be nil therefore there is no need to check for errors, am I right?

Reply:
@arberiii I think I was just being lazy. Should add instrumentation here.